pylintrc

```diff
@@ -14,6 +14,7 @@ max-line-length=131
 
 [TYPECHECK]
 ignored-classes=pytest,lxml.etree
+ignored-modules=distutils
```

Comment:
I'll revert what I did to `install.py` and come back to this in a separate PR. I'm having trouble getting pylint working even on python 2.7 in the latest version, I think they might have a regression.